internal/cloudapi/server.go

```diff
@@ -37,6 +36,20 @@ type Server struct {
 
 type contextKey int
 
+const (
+	identityHeaderKey contextKey = iota
+)
+
+type identityHeader struct {
+	Identity struct {
+		AccountNumber string `json:"account_number"`
+	} `json:"identity"`
+}
```

Comment:
These can just stay in the VerifyIdentityHeader function, less diff noise that way.